docs/api/portable-stories.md

```diff
@@ -0,0 +1,376 @@
+---
+title: 'Portable stories'
+---
+
+export const SUPPORTED_RENDERERS = ['react', 'vue'];
+
+<If notRenderer={SUPPORTED_RENDERERS}>
+
+<Callout variant="info">
+
+Portable stories are currently only supported in [React](?renderer=react) and [Vue](?renderer=vue) projects.
+
+</Callout>
+
+</If>
+{/* End non-supported renderers */}
+
+<If renderer={SUPPORTED_RENDERERS}>
+
+Portable stories are Storybook [stories](../writing-stories/index.md) which can be used in external environments such as in a test (with Vitest, for instance). You can use the [`composeStories`](#composestories) and [`composeStory`](#composestory) functions to compose stories and their [annotations](#annotations), making them portable.
```

Comment:
Seems like it would be better to have separate pages, e.g.
- portable stories in Unit testing
- portable stories in Playwright CT
- portable stories in Cypress CT